packages/fxa-support-panel/test/lib/api.spec.ts

```diff
@@ -76,6 +76,7 @@ function createDefaults(): MockCallsResponse {
     subscriptions: {
       response: [
         {
+          created: 1555354567,
```

Comment:
Could add a test to see if the template is rendered with the correct content.

Reply:
I will do that. Thanks